spec/services/doc_auth/socure/requests/document_request_spec.rb

```diff
@@ -0,0 +1,103 @@
+require 'rails_helper'
+
+RSpec.describe DocAuth::Socure::Requests::DocumentRequest do
+  let(:document_capture_session_uuid) { 'abc123' }
+  let(:redirect_url) { 'https://somewhere.com' }
+  let(:language) { :en }
+
+  subject(:document_request) do
+    described_class.new(document_capture_session_uuid:, redirect_url:, language:)
+  end
+
+  describe 'a new request' do
+    it 'exists' do
+      expect(document_request).to be
+    end
+  end
+
+  describe '#fetch' do
+    let(:language) { :en }
+    let(:document_type) { 'license' }
+    let(:fake_socure_endpoint) { 'https://fake-socure.com/' }
+    let(:socure_document_capture_url) { 'https://verify.socure.us/something' }
+    let(:docv_transaction_token) { 'docv transaction token' }
+    let(:fake_socure_response) do
+      {
+        referenceId: 'socure-reference-id',
+        data: {
+          eventId: 'socure-event-id',
+          customerUserId: '121212',
```

Comment:
```suggestion
          customerUserId: document_capture_session_uuid,
```
we're not testing this here, but if you want, this is what part of what we'd be expecting back